gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/AvatarUpdateResult.kt

```diff
@@ -0,0 +1,42 @@
+package com.gravatar.quickeditor.ui.editor
+
+import android.net.Uri
+import java.util.Objects
+
+/**
+ * All possible results for the Avatar update flow
+ */
+public sealed class AvatarUpdateResult {
+    /**
+     * Quick Editor dismissed without any errors or updates to the Gravatar
+     */
+    public data object Dismissed : AvatarUpdateResult() {
```

Comment:
`data object` doesn't generate `copy()` or `componentN()`  functions so it's safe to use. This can be verified in the `gravatar-quickeditor.api` file.